style/properties/longhands/inherited_text.mako.rs

```diff
@@ -145,55 +145,12 @@ ${helpers.predefined_type(
     affects="layout",
 )}
 
-<%helpers:single_keyword
-    name="white-space"
-    values="normal pre nowrap pre-wrap pre-line"
-    engines="servo",
-    gecko_enum_prefix="StyleWhiteSpace"
-    needs_conversion="True"
-    animation_value_type="discrete"
-    spec="https://drafts.csswg.org/css-text/#propdef-white-space"
-    servo_restyle_damage="rebuild_and_reflow"
```

Comment:
I'm not sure if `servo_restyle_damage` or `needs_conversion` need to be kept.

Reply:
Yes, let's keep these for now. They are used for incremental layout. It's unclear how much of these damage flags we will keep for Layout 2020, but I think we shouldn't modify them as much as possible until we have incremental layout in place.

Reply:
I have added these to both longhands.